### PR TITLE
Add a page on deploying options

### PR DESCRIPTION
**Pull request by bethaitman**
I feel there should be a high-level TOC item on deploying, though I don't know if this is the correct time to do it: as we don't tell you _how_ to deploy with any of these options yet, this might be premature.

Anyway, this is what I think it should look like when we're ready to start having a deployment section. I'm happy to merge it now, or to wait: @meiersi-da, I think that's basically your call.

### CODE REVIEW

**Comment by meiersi-da** [comment]
@bethaitman thanks for suggesting this. As you say, we don't say how to deploy in the section on deployment, which I fear will be off-putting to users. What I'd suggest is to add commentary that explains the current state:
- you can deploy to the Sandbox with Persistence (link to its docs)
- there are more deployment options in development (include the table that we have)

If you could do these changes, then I believe we should merge the page. The TOC entry is a valuable and obvious one to have.

**Comment by bethaitman** [comment]
@meiersi-da: I agree with what you said, and I've made the changes you requested :)

**Comment by meiersi-da** [comment]
Thanks a lot @bethaitman-da . Good changes. From my side, we're good to merge.

File: docs/source/deploy/index.rst [hunk 1]
```diff
@@ -0,0 +1,71 @@
+.. Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Deploying to DAML ledgers
+*************************
+
+To run a DAML application, you'll need to deploy it to a DAML ledger.
+
+How to deploy
+-------------
```
Comment (bame-da) [inline docs/source/deploy/index.rst]:
```suggestion
=============
```

File: docs/source/deploy/index.rst [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+You can deploy to:
+
+- The Sandbox with persistence. For information on how to do this, see the section on persistence in :doc:`/tools/sandbox` docs
+- Further deployment options which are in development. For information on these options and their stage of development, see the table below.
+
+DAML ledgers built or in development
+------------------------------------
```
Comment (bame-da) [inline docs/source/deploy/index.rst]:
```suggestion
====================================
```

**Pull request merged**